src/librustdoc/clean/blanket_impl.rs

```diff
@@ -20,8 +20,9 @@ impl<'a, 'tcx> BlanketImplFinder<'a, 'tcx> {
 
     // FIXME(eddyb) figure out a better way to pass information about
     // parametrization of `ty` than `param_env_def_id`.
```

Comment:
Should this be:

```suggestion
    // parametrization of `ty` than `item_def_id`.
```

?

Reply:
Same as above, I removed the comment.